src/include/duckdb/execution/expression_executor_state.hpp

```diff
@@ -29,7 +29,11 @@ struct ExpressionState {
 	DataChunk intermediate_chunk;
 	string name;
 	double time;
+#ifdef RDTSC
+	CycleCounter profiler;
```

Comment:
The consumer of an interface should never need to use an #ifdef.  Instead, add the ifdef into `cycleclock.hpp`, where using CycleCounter will simply fallback to using Profiler<high_resolution_clock> if RDTSC is not available.

Reply:
Sure!